pkg/syncer/status/status_controller.go

```diff
@@ -85,12 +86,24 @@ func NewStatusSyncer(syncerLogger logr.Logger, syncTargetWorkspace logicalcluste
 			logger.V(2).Info("Set up downstream informer", "gvr", gvr.String())
 			return cache.ResourceEventHandlerFuncs{
 				AddFunc: func(obj interface{}) {
+					unstrob := obj.(*unstructured.Unstructured)
+					if hasDelaySyncAnnotation(unstrob) {
+						logger.V(2).Info("Delay status syncing", "gvr", gvr.String(), "name", unstrob.GetName(), "namespace", unstrob.GetNamespace(), "since annotation is set", storage.DelayStatusSyncing)
+						return
+					}
+
 					c.AddToQueue(gvr, obj, logger)
 				},
 				UpdateFunc: func(oldObj, newObj interface{}) {
 					oldUnstrob := oldObj.(*unstructured.Unstructured)
 					newUnstrob := newObj.(*unstructured.Unstructured)
 
+					// If the object has a delay syncing annotation we do nothing
```

Comment:
In order not to leave an event, we should modify ` deepEqualFinalizersAndStatus` in order to also compare the value of the value of the delaying annotation, so that the removal of the annotation would enqueue of the resource again.